llvm/lib/MCA/InstrBuilder.cpp

```diff
@@ -749,7 +759,7 @@ InstrBuilder::createInstruction(const MCInst &MCI,
     RegID = WD.isImplicitWrite() ? WD.RegisterID
                                  : MCI.getOperand(WD.OpIndex).getReg();
     // Check if this is a optional definition that references NoReg.
```

Comment:
nit: update comment?
maybe to `Check if this is a optional definition that references NoReg or writes to a constant register`

Reply:
Good point, updated the comment.